Tapptitude/Helpers/TTTableViewUpdater.swift

```diff
@@ -8,8 +8,31 @@
 
 import UIKit
 
-protocol TTTableViewUpdater {
-    
+public class TTRowAnimationConfig {
```

Comment:
Maybe a structure is better ?

public structure TTRowAnimationConfig {
  public var itemsReload: UITableView.RowAnimation = .automatic
....
}